src/sentry/api_gateway/proxy.py

```diff
@@ -0,0 +1,51 @@
+"""
+Utilities related to proxying a request to a region silo
+"""
+
+
+from django.http import StreamingHttpResponse
+from requests import Response as ExternalResponse
+from requests import request as external_request
+from rest_framework.exceptions import NotFound
+from rest_framework.request import Request
+
+from sentry.models.organization import Organization
+
+# stream 0.5 MB at a time
+PROXY_CHUNK_SIZE = 512 * 1024
+
+
+def _parse_response(response: ExternalResponse) -> StreamingHttpResponse:
+    """
+    Convert the Responses class from requests into the drf Response
+    """
+    streamed_response = StreamingHttpResponse(
+        streaming_content=(chunk for chunk in response.iter_content(PROXY_CHUNK_SIZE)),
+        status=response.status_code,
+        content_type=response.headers.pop("Content-Type"),
+    )
+    # Add Headers to response
+    for header, value in response.headers.items():
+        streamed_response[header] = value
```

Comment:
Couldn't find a better way to add the header.

Reply:
Seems okay to me.